app/config/security.yml

```diff
@@ -287,7 +288,8 @@ security:
         - { path: ^/profile/jwt, roles: IS_AUTHENTICATED_REMEMBERED }
         - { path: ^/profile/notifications, roles: IS_AUTHENTICATED_REMEMBERED }
         - { path: ^/profile, roles: IS_AUTHENTICATED_REMEMBERED }
-        - { path: ^/admin, roles: ROLE_ADMIN }
+        - { path: ^/admin/settings, roles: ROLE_ADMIN }
+        - { path: ^/admin/.*$, roles: ROLE_DISPATCHER }
```

Comment:
Actually `^/admin` should be accessible to `ROLE_DISPATCHER`, because the button to access admin interface points to this URL, and it needs to be maintained even if it redirects to a default page. 

It's convenient to just type `/admin` (this is what I do 🙂 )

<img width="406" alt="Capture d’écran 2023-03-20 à 15 26 46" src="https://user-images.githubusercontent.com/1162230/226370433-6fe88750-c5ae-41a4-8577-583ec3201205.png">